src/pages/Movie/index.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Movie() {
```

Comment:
This is a placeholder for the trailer page right?

Reply:
Yes